packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -438,11 +438,15 @@ export const run = async () => {
       try {
         await func()
       } catch (err) {
-        logger.error('Error submitting batch', {
```

Comment:
I feel like the status should be added to the object instead of being placed in string to allow for searching on subkeys more easily. I'm not an expert on sentry but my intuition is that you can get all errors with an exact match of `Error submitting batch` and then filter based on the values in the object and then also more easily create things like pie charts that show the percentages of the different statuses

Reply:
might be what we have to do since it seems like the errors both triggered as we submit the transaction

Reply:
i left the status in one log for server errors so we can distinguish between the different ones, since sentry categorizes these errors by log line and doesn't have a way to filter by the rest of the error object